docker-compose.ac.yml

```diff
@@ -180,15 +180,15 @@ services:
       TRIGGER_EDGEXMESSAGEBUS_PUBLISHHOST_HOST: mqtt-broker
       TRIGGER_EDGEXMESSAGEBUS_SUBSCRIBEHOST_HOST: mqtt-broker
       APPLICATIONSETTINGS_AUTHENTICATIONENDPOINT: http://ms-authentication:48096/authentication
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDDISPLAYRESET: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/displayReset
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDDISPLAYROW0: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/displayRow0
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDDISPLAYROW1: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/displayRow1
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDDISPLAYROW2: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/displayRow2
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDDISPLAYROW3: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/displayRow3
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDLOCK1: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/lock1
-      APPLICATIONSETTINGS_DEVICECONTROLLERBOARDLOCK2: http://edgex-core-command:48082/api/v2/device/name/ds-controller-board/command/lock2
-      APPLICATIONSETTINGS_INFERENCEDOORSTATUS: http://edgex-core-command:48082/api/v2/device/name/Inference-MQTT-device/command/inferenceDoorStatus
-      APPLICATIONSETTINGS_INFERENCEHEARTBEAT: http://edgex-core-command:48082/api/v2/device/name/Inference-MQTT-device/command/inferenceHeartbeat
+      APPLICATIONSETTINGS_CONTROLLERBOARDDISPLAYRESETCMD: displayReset
+      APPLICATIONSETTINGS_CONTROLLERBOARDDISPLAYROW0CMD: displayRow0
+      APPLICATIONSETTINGS_CONTROLLERBOARDDISPLAYROW1CMD: displayRow1
+      APPLICATIONSETTINGS_CONTROLLERBOARDDISPLAYROW2CMD: displayRow2
+      APPLICATIONSETTINGS_CONTROLLERBOARDDISPLAYROW3CMD: displayRow3
+      APPLICATIONSETTINGS_CONTROLLERBOARDLOCK1CMD: lock1
+      APPLICATIONSETTINGS_CONTROLLERBOARDLOCK2CMD: lock2
+      APPLICATIONSETTINGS_INFERENCEDOORSTATUSCMD: inferenceDoorStatus
+      APPLICATIONSETTINGS_INFERENCEHEARTBEATCMD: inferenceHeartbeat
```

Comment:
These can be removed since that are the same values as what is in the TOML file. 

They were here previously since the host name was in the value.